frappe/public/less/quill.less

```diff
@@ -131,13 +131,28 @@
 
 .ql-editor ol {
 	padding-left: 2.5em;
+  padding-right: 2.5em;
```

Comment:
```suggestion
	padding-right: 2.5em;
```

Reply:
Notes have been fixed.

Reply:
I don't know why it dos not work withe me I'm using Atom editor and I use tabs for indent !!